### PR TITLE
minor rewordings

### PR DESCRIPTION
**Pull request by gvdr**
Just some minor typos and some rephrasing to result more readable.

### CODE REVIEW

**Comment by aammd** [comment]
Helpful! Thank you @gvdr 😄

**Comment by aammd** [comment]
@gvdr Uh -oh it looks like you *do* have merge conflicts after all! I'll merge this PR once you bring in those changes

**Comment by gvdr** [comment]
As for #139 I added a rewording of the bit about *under the hood* of data.frames.

File: 04-data-structures-part1.Rmd [hunk 1]
```diff
@@ -310,7 +310,34 @@ We can now understand something a bit surprising in our data.frame; what happens
 typeof(cats)
 ```
 
-We see that data.frames look like lists 'under the hood' - this is because a data.frame is really a list of vectors and factors, as they have to be - in order to hold those columns that are a mix of vectors and factors, the data.frame needs something a bit more flexible than a vector to put all the columns together into a familiar table.
+Under the hood, a data.frame is a list of variables, which can be of different kind. In our `cats` example, we have an integer, a double and a logical variable. As we habe seen already, ach column of data.frame is a vector.
```
Comment (tomwright01) [inline 04-data-structures-part1.Rmd]: Couple of types habe/have ach/each

**Comment by gvdr** [comment]
Solved the ach habe typos.

**Pull request merged**